main/activity-log.js

```diff
@@ -63,7 +63,7 @@ class ActivityLog {
  */
 function loadStoredEntries () {
   // A workaround to fix false TypeScript errors
-  return /** @type {any} */(activityLogStore.get('events', []))
+  return /** @type {any} */(activityLogStore.get('activities', []))
```

Comment:
This bug was tricky to find 🤓

Reply:
I think this was the reason why `ActivityLog.reset()` was not working in our pair-programming session, @juliangruber .